### ISSUE TITLE
Include fees in the balance that left a wallet listed on transaction list

### ISSUE THREAD

**Issue by stackingsaunter** [getAlby/hub#464]
Currently, they are only visible in details.

**Comment by bumi** [comment]
do you really want that? more numbers? what will it tell people and which other tranasction lists do that?

**Comment by stackingsaunter** [comment]
How is it more numbers? It's the same amount of numbers

Wallet that do that: Phoenix, Bitkey, Strike, Bitkit, Blink, Aqua

Wallets that don't do that: Muun, Blockstream Green, Umbrel

**Comment by rolznz** [comment]
The title of this issue is weird, but this is what you mean, right?

Use the sum of the amount + fee for the value shown on entries in the transaction list

Then when you open the details you would see the amount + fee separated

**Comment by rolznz** [comment]
@reneaaron brought up a good point - if we make this change it should be consistent across all Alby products (currently none are done this way)

**Comment by GBKS** [comment]
I agree with @rolznz. Show the total amount that left your wallet in the transaction list, and show the breakdown in the details (maybe also in a hover tool tip on desktop).

**Comment by stackingsaunter** [comment]
Thanks @GBKS for your input, I also think this is best

**Comment by stackingsaunter** [comment]
In that case I would adopt this in all our products, and we could start with Hub? @rolznz

**Comment by Aditya-PS-05** [comment]
@rolznz , can you assign me this issue?

**Comment by rolznz** [comment]
@Aditya-PS-05 feel free to submit a PR, thanks!
